UI/window-projector.cpp

```diff
@@ -848,8 +848,51 @@ void OBSProjector::mousePressEvent(QMouseEvent *event)
 					SLOT(OpenWindowedProjector()));
 
 		} else if (!this->isMaximized()) {
+
+			auto resizeToCustomScaleAction = [this](double scale) {
+				ResizeToScale(scale);
+			};
+
```

Comment:
```suggestion
```
This function is unnecessary as-is, but unneeded with the other changes I list below.